lib/client/websocket-tracker.js

```diff
@@ -122,7 +127,10 @@ WebSocketTracker.prototype._openSocket = function () {
   self.socket = socketPool[self.announceUrl]
   if (!self.socket) {
     self.socket = socketPool[self.announceUrl] = new Socket(self.announceUrl)
+    self.socket.consumers = 1
     self.socket.on('connect', self._onSocketConnectBound)
+  } else {
+    socketPool[self.announceUrl].consumers++
```

Comment:
Reverse the conditional. Now that there's another case, it's best to avoid `!` whenever possible.